scripts/nord_login

```diff
@@ -17,6 +17,8 @@ then
 fi
 
 ## only token is viable in a container
+[[ -z "${TOKEN}" ]] && [[ -f "${TOKEN_FILE}" ]] && TOKEN="$(head -n 1 "${TOKEN_FILE}")"
```

Comment:
We need to declare `TOKEN_FILE` as /run/secrets/token_file (static declaration before the evaluation), then describe in`README` that `TOKEN_FILE` must be declared as a secret in order to be leveraged:
```
secrets:
   token_file:
     file: token.txt

Reply:
The TOKEN_FILE env var is enough to allow use of Docker Compose's secrets. In the other hand, declarable env var gives more control on how and where this file is delivered to the filesystem. Look at my docker compose yaml for working example:
```yaml
services:
  vpn:
    image: nordvpn-custom:latest # just the same image with my own changes
    container_name: vpn
    cap_add:
      - NET_ADMIN
    secrets:
      - nordvpn_token
    environment:
      - TOKENFILE=/run/secrets/nordvpn_token
      - TECHNOLOGY=OpenVPN
      - PROTOCOL=TCP
      - OBFUSCATE=TRUE
      - CONNECT=pl206
      - NETWORK=192.168.0.0/24  # So it can be accessed within the local network
    sysctls:
      - net.ipv6.conf.all.disable_ipv6=1  # Recomended if using ipv4 only
secrets:
  nordvpn_token:
    file: "/path/to/token"
```

Reply:
Hmm, that shouldn't work.  `TOKENFILE=/run/secrets/nordvpn_token` should be `TOKEN_FILE=/run/secrets/nordvpn_token`  for `$TOKEN_FILE` to have a value.

I hear you on flexibility, but I think most people are not savvy on how these things work and it would be easier for them if they only needed to declare secrets > token_file > file and not have to also declare the path to the secret in the container as an ENV.  Hope that makes sense.

I think this is a good addition to the project, we just need to keep it clean and as simple as is practical.

Reply:
> should be TOKEN_FILE

Yep, just a mistake

> keep it clean and as simple as is practical

Agree